### DISCUSSION TITLE
Add 'bug report' template

### DISCUSSION THREAD

**Discussion by BraINstinct0** [valinet/ExplorerPatcher#143] [Ideas]
Obviously many stray, duplicate issues(including mine) and feature requests are constantly coming up on the Issues section.
Adding an issue template that tells you to 'look up for duplicates' might help. Also it'll be easier for ppl to write more concise reports(by mentioning ExplorerPatcher&Windows version explicitly, etc.).

**Comment by valinet** [comment]
C'mon, let's stop kidding ourselves, no matter how many warnings you put up, some people will still disregard them or post issue descriptions that bring memories of the SMS era. Isn't it plenty obvious already that duplicate issues should be avoided...? Idk, if you want to create a template, sure, idk, we can include that I guess... but I am busy with some other stuff atm, don't have time to think about it right now.

**Reply by BraINstinct0** [reply]
>if you want to create a template, sure, idk, we can include that I guess...
Will do it soon™! (which, might not be so soon)
I agree that it won't stop most from dupe reporting tho..

**Reply by BraINstinct0** [reply]
Aand I made a simple template.
https://github.com/valinet/ExplorerPatcher/pull/163

**Reply by Poopooracoocoo** [reply]
I think you should reconsider adding an issue template. It's the perfect opportunity to block blank issues and force people to read not to create issues. It's not going to deter _everyone_ but it should help, right? You can also present a bug report template so feature requests don't fit, automatically directing people to discussions for questions or feature requests. :)

**Reply by thatsofia** [reply]
@valinet You should consider adding a template. 

I'll also take this comment to thank you. I've seen how tirelessly you support the community who doesn't even give back. It's incredible.
